.env.example

```diff
@@ -3,6 +3,9 @@
 # Replace the following with the output from "rake secret"
 APP_SECRET_TOKEN=REPLACE_ME_NOW!
 
+# Replace the following with another output from "rake secret"
+APP_ENCRYPTION_PASSPHRASE=REPLACE_ME_NOW!
```

Comment:
Agreed. I was thinking about defaulting it to `APP_SECRET_TOKEN`, and telling the user to set `APP_ENCRYPTION_PASSPHRASE` if they ever change `APP_SECRET_TOKEN`? So `encryption_key = ENV['APP_ENCRYPTION_PASSPHRASE'].presence || ENV['APP_SECRET_TOKEN']`